packages/cozy-sharing/src/SharingBanner/hooks/useSharingInfos.jsx

```diff
@@ -4,7 +4,7 @@ import { useClient, models } from 'cozy-client'
 
 import getQueryParameter from '../helpers/QueryParameter'
 import logger from '../../logger'
-import { buildSharingsByIdQuery } from '../helpers/queries'
+import { buildSharingsByIdQuery } from '../../queries/queries'
```

Comment:
I think we want to avoid relative imports

Reply:
> I think we want to avoid relative imports

This is not really true and this is not really false neither. 

We need relative import if the component needs to be overridden. For now, this lib is not concerned, yet. 

Also, we're within a lib, so in order to be able to do that, we need to transform the path only at the transpilation/publish time. (see https://github.com/cozy/cozy-ui/blob/master/babel.config.js#L35-L36 for instance)

Since this is not the scope of this PR, let's move on ;)

Reply:
Thank you for the answer!